.pre-commit-config.yaml

```diff
@@ -17,13 +17,6 @@ repos:
     - id: check-yaml
     - id: check-added-large-files
 
-# Shellcheck
-- repo: https://github.com/jumanjihouse/pre-commit-hooks
-  rev: 2.1.5
```

Comment:
Something wrong with this?

Reply:
@m-kuhn , I couldn't get it to commit on my machine, I'll revert later. I kept getting this:

```
Test shell scripts with shellcheck.......................................Failed
- hook id: shellcheck
- exit code: 1

This check needs shellcheck from https://github.com/koalaman/shellcheck

```